Chem Additions - Cooling and Freezing

LordZirconium opened the pull request:
## About the PR
+ Added Cooling reaction
+ Made Cryo Chemicals and Cognizine now require super-cool temperatures to make. 
+ Fresium now has more uses! You can use it as replacement (Use with Nitrogen) for FrostOil in solidifying materials reactions.
## Why / Balance
Useful, maybe gonna make another PR for solidifying Steel and Plasteel.
Gives Atmosians more reasons to make frezon and interact with station's crew ( Basically fixing problem that salv once had, we'll see how it goes. )
And just adding more cool reactions to the game, which is always fun!
Also I stumbled on Opporozidone reaction and it had this comment:
**#Maybe if a method of reducing reagent temp exists one day, this could be -50**
Now it is! ( Kinda, I set it to -150 to present chem with more challenge. )
## Technical details
Recipes:
Fresium x10 + Nitrogen x10 + Silver x10 = Silver Ingot
Fresium x10 + Nitrogen x10 + Gold x10 = Gold Ingot
Fresium x20 + Nitrogen x20 + Plasma x10 = Plasma Sheet
Cooling:
Fresium x4 + Ammonia x2 + Chlorine x1 + Water x3 = Nothing and Lowers solution temperature by 25K, scaleable.
( Was based on some 6th grade endothermic reaction of Ammonia Chloride ( NH4Cl + HO2 ) and fresium is in there just because I want to give it more uses. ) 

##  Media
**( Coming right up! )**
## Requirements
- [ X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ ] I have added media to this PR or it does not require an ingame showcase. **( Not yet, but soon! )**

**Changelog**
<!--
:cl:
- add: You can now solidify plasma!
- add: Added cooling reaction! More frezon uses!
- tweak: Made Cryo chemicals require cooler temperatures!
-->

ViceEmargo commented:
I like the concept of cryo chemicals requiring cold temperatures in order to manufacture, however, gating cryo chemicals behind Fresium at this time will birth the death of all cryogenic healing.

Atmos techs were already given less and less reason to make Frezon, and forcing them to spend the time making that will be a net negative on the interaction between atmos and medbay.
We need a function of cooling reagents in a container without the use of this reaction to honestly justify making cryo chems have that requirement to make, and even then, the current set of Cryo chems arn't so powerful that they should be gated behind a difficult to obtain reagent such as Fresium to create a reaction to achieve incredible low temperatures.

LordZirconium commented:
> I like the concept of cryo chemicals requiring cold temperatures in order to manufacture, however, gating cryo chemicals behind Fresium at this time will birth the death of all cryogenic healing.
> 
> Atmos techs were already given less and less reason to make Frezon, and forcing them to spend the time making that will be a net negative on the interaction between atmos and medbay. We need a function of cooling reagents in a container without the use of this reaction to honestly justify making cryo chems have that requirement to make, and even then, the current set of Cryo chems arn't so powerful that they should be gated behind a difficult to obtain reagent such as Fresium to create a reaction to achieve incredible low temperatures.

Absolutely, but I won't change the reaction.
So uhh Imma add new one that is SLIGHTLY debuffed, you know, would that be cool with ya?

Sparlight commented:
have you ever made frezon in atmos and  then actually condensed it
you won't get any significant amount of condensed frezon by the time the shift ends

I like where your head is at but I feel like unless we have a veritable rainbow of reactions that cool beakers instead of heat them, we need some doodad that makes a beaker cold.  Like I could either spend 30 seconds on the hotplate or just make some chloral hydrate to skip ever needing the hotplate.  But the hotplate is there for me.

I agree with the sentiment that having cryo chems gated behind this will be the death knell of cryo as we know it.  It's already significantly underused despite how powerful it is, and I don't see it being used almost at all with this.  Would be cool if we had more reactions for novel chems that needed cooling though.

ArtisticRoomba commented:
Cryochems requiring Fresium is an *extremely* big killer to any cryomeds. It is unfortunately not very common for Atmospherics to produce Frezon at all and gating an entire subsection of content behind it is not a good idea.

This is especially considering the fact that Frezon still needs to be rebalanced to make it mildly more accessible to atmospherics. Yeah I'll get around to it soon but I'm busy.

LordZirconium commented:
> I think the amount of fresium used to solidify stuff should be lowered, it's fairly difficult to make compared to what you're getting. Chemists are highly unlikely to part with that much in exchange for materials that cargo or salvage can obtain relatively easily. Maybe make fresium a catalyst for these reactions?

I Implemented fresium in early stages of this PR, Yeah, I should bring fresium catalyst back... Though it seems unbalanced? It would be like Goob's cryostalane in some way... but yeah, good idea in general.

LordZirconium commented:
And yeah, I noticed fresium being reqiurement complaints, I am working on it, no sweat, cryo is not dead yet.

LordZirconium commented:
_Ehhh... I'll fix it tomorrow_

LordZirconium commented:
Fixed! Got it working, pretty cool.
Added 3 more endothermic reactions, not a rainbow boquette as requested but honestly, I will make second part of this PR that will feature those **already mapped** Freezers. For now, have some Alkali-Silicon reactions.

LordZirconium commented:
_I got outdated before even getting merged_

LordZirconium commented:
_NICE, HELL YEAH, THAT SOLVED IT!_
Bruh... how do I remove conflict tag.

LordZirconium commented on Resources/Prototypes/Recipes/Reactions/medicine.yml:
Bruh, traumaxadone, who thought of that?

LordZirconium commented:
agony

august-sun commented:
Hey, wanted to follow up here in terms of solidification: Would also be neat if Trit + Frezon could be condensed and reacted to solidify into Uranium. The previously existing problems of Frezon being inaccessible are still a thing to be unravelled for sure, but figured it'd be good to include with this!
